db/model/auditevent.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/auditevent.js
+ *
+ */
+const constants = require('../constants');
+
+module.exports = function auditevent(seq, dataTypes) {
+  const AuditEvent = seq.define('AuditEvent', {
+    id: {
+      type: dataTypes.UUID,
+      primaryKey: true,
+      defaultValue: dataTypes.UUIDV4,
+    },
+    loggedAt: {
+      type: dataTypes.BIGINT,
+      allowNull: false,
```

Comment:
I'm thinking it might make sense to allow this to be null but when it's null to default it to current time. What do you think?